examples/experimental/Length of Stay demo on MIMIC/pytorch/trainer.py

```diff
@@ -0,0 +1,121 @@
+"""
+Module containing functions useful for training a pytorch neural network
+"""
+import random
+import numpy as np
+import logging
+import torch
+import torch.optim as optim
+import torch.nn.functional as F
+
+
+from pytorch.models import FeedforwardNeuralNetwork
+
+# The loss functions used must be averaged over the batch to be consistent with
+# the calculation of the metrics.
+LOSSES = {"mean_squared_error": F.mse_loss}
+
+OPTIMIZERS = {"rmsprop": optim.RMSprop}
+
+logger = logging.getLogger(__name__)
+
+
+def train(
+    args: object,
+    train_dataloader: object,
+    val_dataloader: object,
+    n_features: int,
+    hidden_dim: int = None,
+    final_activation: str = "linear",
+    loss_name: str = "mean_squared_error",
+    nb_epoch: int = 250,
+    learning_rate: float = 0.005,
+    ffn_depth: int = 2,
+    optim_name: str = "rmsprop",
+) -> object:
+    """performs the training of FeedforwardNeuralNetwork
+
+    Args:
+        args: an instance of Arguments
+        train_dataloader : an instance of the torch DataLoader class containing
+            the training data
+        val_dataloader : an instance of the torch DataLoader class containing
+            the validating data
+        n_features : dimension of an example, i.e. the number of feature
+        hidden_dim : hidden layer size. Defaults to None.
+        final_activation : activation function's name of the last layer. The
+                possible choices are listed in ACTIVATION_FUNCTION. Defaults to
+                "linear".
+        loss_name : loss' name. The possible choices are listed in LOSSES.
+            Defaults to "mean_squared_error".
+        nb_epoch : number of epochs. Defaults to 250.
+        learning_rate : learning rate. Defaults to 0.005.
+        ffn_depth : number of hidden layers. Defaults to 2.
+        optim_name : optimizer's name. The possible choices are listed in
+            OPTIMIZERS. Defaults to "rmsprop".
+    Return:
+        the trained model
+    """
+    set_seed(args)
```

Comment:
Can you directly provide args.seed to the function?

Reply:
I've made the changes